src/Symfony/Component/Finder/Finder.php

```diff
@@ -614,6 +614,20 @@ public function append($iterator)
     }
 
     /**
+     * Check if the any results were found.
+     *
+     * @return bool
+     */
+    public function hasResults()
+    {
+        foreach ($this->getIterator() as $null) {
```

Comment:
Minor suggestion: `$null` may be confusing and in the future, it may be a reserved word in PHP. Could we use `$_` as the name of this unused variable as we do in other parts of Symfony? Thanks!

Reply:
Sure, I wasn't aware of the convention, I've updated it to use `$_` now